config/locales/en.yml

```diff
@@ -2839,11 +2841,14 @@ en:
   text_work_packages_ref_in_commit_messages: "Referencing and fixing work packages in commit messages"
   text_journal_added: "%{label} %{value} added"
   text_journal_changed_html: "%{label} changed from %{old} %{linebreak}<strong>to</strong> %{new}"
+  text_journal_changed_html_project_activity: "%{label} changed from %{old} %{linebreak}to %{new}"
```

Comment:
You implemented it as specified, but I wonder if that "to" shouldn't be bold like in other strings.
I see no good reason to be inconsistent here so I asked the question to @psatyal in elements.

Reply:
So Parimal's answer is that for consistency the `to` should not be bold. So `text_journal_changed_html_project_activity` key should be removed, and `text_journal_changed_html` key should be updated to change the `<strong>to</strong>` into `to`.